CHANGELOG.md

```diff
@@ -25,18 +29,18 @@
   * Cleanup and make semantic token processing and testing code more consistent (PR: [#77684](https://github.com/dotnet/roslyn/pull/77684))
   * Fix generation of attribute with array constant (PR: [#77661](https://github.com/dotnet/roslyn/pull/77661))
 * Bump Razor to (PR: [#8096](https://github.com/dotnet/vscode-csharp/pull/8096))
-  * Some code actions ordering and cleanup (#11659) (PR: [#11659](https://github.com/dotnet/razor/pull/11659))
-  * Emit pragmas for empty @ expressions in MVC (#11653) (PR: [#11653](https://github.com/dotnet/razor/pull/11653))
-  * Couple of small code actions tweaks (#11630) (PR: [#11630](https://github.com/dotnet/razor/pull/11630))
-  * Fix text area formatting in the new formatting engine (#11624) (PR: [#11624](https://github.com/dotnet/razor/pull/11624))
+  * Some code actions ordering and cleanup (PR: [#11659](https://github.com/dotnet/razor/pull/11659))
```

Comment:
@ryzngard For # numbers that would be valid in this repo they get replaced with links to this repo. This bug is fixed in the latest roslyn-tools.